people/stepan

```diff
@@ -0,0 +1,12 @@
+{
+ "key": "stepan",
+ "name": "Bc. Štěpán Karel",
+ "nick": "Štěpán",
+ "email": "stepan.karel@mild.blue",
+ "title_cs": "Tester",
+ "title_en": "Tester",
+ "title_de": "Tester",
+ "short_desc_en": "Štěpán helps in Mild Blue with aplications doing what they was designed for a to be as user friendly as they can be.",
```

Comment:
Tady vidím trochu gramatiky špatně. Lépe bude znít: Štěpán helps at Mild Blue with applications to do what they were designed to do, to be as user-friendly as they can be.